app/src/components/ShipmentForm/DetailsForm.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+	import { createForm } from 'felte';
+	import { reporter, ValidationMessage } from '@felte/reporter-svelte';
+	import Button from '../Buttons/Button.svelte';
+	import Details from './Details.svelte';
+	import { validator } from '@felte/validator-yup';
+	import * as yup from 'yup';
+	import type { DetailsFormInterface } from './interfaces';
+
+	export let initialValues: DetailsFormInterface;
+	export let onSubmit;
+	export let onBack;
+
+	export let showModal = true;
+
+	const schema = yup.object({
+		count: yup.number().required(),
+		access: yup.string().transform(parseInt).required(),
+		priority: yup.number().required(),
+		fragility: yup.number().required()
+	});
+
+	const { form, data } = createForm<yup.InferType<typeof schema>>({
```

Comment:
Same with validation.

Reply:
hmm, i would do it but i want to first resovle string/integer parsing, between felte library and dom, cause now theres too many strings